components/Landing/HeroMoment/LandingHeroMomentComponent.vue

```diff
@@ -123,17 +211,31 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
 
   &__title {
     margin-bottom: carbon.$spacing-06;
+    font-weight: 500;
+    font-size: 5rem;
+    display: flex;
+    align-items: center;
 
     // TODO: Force pointer events to allow the user to select text. Remove
     // when decomissioning the Metal page.
     pointer-events: auto;
 
     @include carbon.breakpoint-up(md) {
       margin-bottom: carbon.$spacing-07;
+      font-size: 7rem;
```

Comment:
We should use a font definition from the reusable defined variables that we get from the Carbon library, like we do in the rest of the codebase.